packages/engine/src/mocap/MotionCaptureSystem.ts

```diff
@@ -130,8 +131,8 @@ const execute = () => {
   for (const entity of motionCaptureQuery()) {
     const peers = Object.keys(network.peers).find((peerID: PeerID) => timeSeriesMocapData.has(peerID))
     const rigComponent = getComponent(entity, AvatarRigComponent)
-    if (!rigComponent.normalizedRig) continue
-    const worldHipsParent = rigComponent.normalizedRig.hips.node.parent
+    if (!rigComponent.bonesToEntities) continue
```

Comment:
this will always be truthy, maybe we should be checking for `rigComponent.bonesToEntities.hips` instead?

Reply:
Yea good point, missed these